src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-install.mdx

```diff
@@ -112,9 +111,7 @@ If you're not using the guided install, follow the instructions for your environ
     id="windows-install"
     title="Windows"
   >
-    1. Download the `nri-jmx` .MSI installer image from:
-
-       [https://download.newrelic.com/infrastructure_agent/windows/integrations/nri-jmx/nri-jmx-amd64-installer.exe](https://download.newrelic.com/infrastructure_agent/windows/integrations/nri-jmx/nri-jmx-amd64-installer.exe)
```

Comment:
For docs folks: no idea if it is better to show the url when downloading something or not 😄 
I followed the same approach we had for kafka, but the old one might be better